app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -21,16 +21,18 @@ def self.rrd_claim_processed?(submission)
 
     # Fetch all claims from EVSS and return whether there are any open EP 020's.
     # This method could be moved into a Concern when ProcessorSelector adds new job classes.
-    def self.pending_eps?(auth_headers)
-      all_claims = EVSS::ClaimsService.new(auth_headers).all_claims.body
-      all_claims['open_claims'].any? { |claim| claim['base_end_product_code'] == '020' }
+    def self.pending_eps?(form526_submission)
+      all_claims = EVSS::ClaimsService.new(form526_submission.auth_headers).all_claims.body
+      pending = all_claims['open_claims'].any? { |claim| claim['base_end_product_code'] == '020' }
+      add_metadata(form526_submission, offramp_reason: 'pending_ep') if pending
```

Comment:
Not that it matters here but out of curiosity how would this behave in the context of a Sidekiq job?

Tiny nit, but is there a style convention for referencing class methods inside the class in which they're defined? I would almost prefer `self.add_metadata(...)`...